testfixtures/tests/test_logcapture.py

```diff
@@ -25,6 +25,52 @@ def test_simple(self):
         root.info('after')
         assert str(l) == "root INFO\n  during"
 
+    def test_simple_strict(self):
+        log_capture = LogCapture(fail_on_error_log=True)
+        root.error('during')
+        log_capture.uninstall()
+        try:
```

Comment:
Use `ShouldAssert` please, and make sure you assert about the error message shown.